setup.py

```diff
@@ -14,6 +18,17 @@
     else:
         reqs += [ir]
 
+
+opt_reqs = []
+opt_links = []
+for ir in optionals:
+    if ir[0:3] == 'git':
+        opt_links += [ir + '#egg=' + ir.split('/')[-1] + '-0']
```

Comment:
it doesn't really matter in the setup file, but this is **not** the format we use in Devito to build/concatenate strings

Reply:
Thanks, what was supposed to be used my bad